timed/reports/views.py

```diff
@@ -226,10 +226,13 @@ def _create_workreport(self, from_date, to_date, today, project, reports, user):
 
             hours = report.duration.total_seconds() / 60 / 60
             table["B13"] = Cell(hours, style_name=float_style)
-
-            table["C13"] = Cell(report.user.get_full_name(), style_name=text_style)
-            table["D13"] = Cell(report.task.name, style_name=text_style)
-            table["E13"] = Cell(report.comment, style_name=text_style)
+            if report.not_billable:
+                table["C13"] = Cell("nein", style_name=text_style)
```

Comment:
Since we're doing all reports in english now anyway, I'd suggest using either "yes"/"no" or something neutral (like placing an "X" mark)

Reply:
Agree, but make sure it's consistent with the naming of the column. In which case a switch of logic would be necessary.

`not report.not_billable` -> "X" mark

Reply:
I have to translate the work report to english anyways so I'll do it in another MR.